### PR TITLE
Add in ability to give negative points. #13

### PR DESCRIPTION
**Pull request by JacobJerrell**
Added NEG_POINTS = 'ALLOW' / 'DISALLOW' environment variable.
If set to allow, it will expect the NEGATIVE_EMOJI, and NEGATIVE_POINTS environment variables.

- Added _extract_type to FireballMessage which will return EMOJI or NEGATIVE_EMOJI
- Added kind to check_user_points_remaining functions ( check_user_points_remaining(user_id, num, kind==EMOJI) ). Instead of coding another function to check negative points. Can we do things this way, or make separate functions for everything?
- Added command handling to the bot

- [ ] Work out leaderboard issues
- [ ] Add negative points support to check_points bot commands
- [ ] Add functionality to Azure/Redis storage
- [ ] Other things I didn't think of

Basically hoping for a code review to make sure I'm headed in the right direction

### CODE REVIEW

File: hey_fireball.py [hunk 1]
```diff
@@ -19,25 +19,33 @@
 BOT_ID = os.environ.get("BOT_ID")
 EMOJI = os.environ.get('EMOJI')
 POINTS = os.environ.get('POINTS')
-SELF_POINTS = os.environ.get('SELF_POINTS', "DISALLOW")
+SELF_POINTS = os.environ.get('SELF_POINTS', "ALLOW")
```
Comment (MattPitlyk) [inline hey_fireball.py:22]: I think the default should be DISALLOW.

File: hey_fireball.py [hunk 2]
```diff
@@ -19,25 +19,33 @@
...
+NEG_POINTS = os.environ.get('NEG_POINTS', "ALLOW")
```
Comment (MattPitlyk) [inline hey_fireball.py:23]: I think the default should be DISALLOW.

Comment (JacobJerrell) [reply hey_fireball.py:23]: Agreed. I must have forgotten to switch those off when I was done testing
